### ISSUE TITLE
Default Boddy Mapping Template for x-www-form-urlencoded: Can it be removed?

### ISSUE THREAD

**Issue by marcelog** [serverless/serverless#3280]
Hello,

I'm defining my API Gateway event like this:
```yaml
service: myService
provider:
 ...
custom:
  ...
  standardRequest:
    passThrough: NEVER
    template:
      "application/json": ${file(doc/templates/generic_request.vtl)}

functions:
  fun:
    ...
    events:
      - http:
          method: post
          integration: lambda
          request: ${self:custom.standardRequest}
```

This is working fine, but when deploying, there are 2 Body Mapping Templates. 1 for `application/json` (great!) and one for `application/x-www-form-urlencoded`, which I understand is one of the defaults provided with this integration, but not one that I'd like to support in my code.

I though that by overwriting the `request` option, and leaving only the support for `application/json` I could get rid of `application/x-www-form-urlencoded`, but that was not the case.

Is this a bug or a feature? How can I avoid supporting this content type?

Thanks in advance!

**Comment by pmuens** [comment]
Hey @marcelog thanks for the question.

This is intended behavior. However you should be able to overwrite this with the help of the `resources` section: https://serverless.com/framework/docs/providers/aws/guide/resources/ in your `serverless.yml` file.

**Comment by arindamghoshal** [comment]
Hi @pmuens, Thanks for the great framework.

I also faced the same issue as @marcelog when I only wanted to configure the Body Mapping Template for "application/json". I needed to exclude the "application/x-www-form-urlencoded" from the "RequestTemplates" object.

I did a fork of the project and made the following change to exclude "application/x-www-form-urlencoded" from the "RequestTemplates" object.

https://github.com/arindamghoshal/serverless/commit/810c75ac21a176b677ee0518a0eaf97a9da57944

Best Regards
Arindam
